_config.yml

```diff
@@ -97,7 +97,7 @@ share-links-active:
 
 # How to display the link to the website in the footer
 # Remove this if you don't want a link in the footer
-url-pretty: "solidity.ethereum.org"  # eg. "deanattali.com/beautiful-jekyll"
+url-pretty: "soliditylang.org"  # eg. "deanattali.com/beautiful-jekyll"
```

Comment:
blog.soliditylang.org

Reply:
yes, but doesn't work yet. :(

Reply:
should we leave it as solidity.ethereum.org until then?

Reply:
Well I would rather not change this until it is working :)